### PR TITLE
feat(mongodb): allow mongodb version 4.2.x

### PR DESCRIPTION
**Pull request by pintojeremy**
# PR Description

Allowing mongodb version 4.2 to be installed using our mongo related ansible role !

I added a bit of idempotence by uninstalling mongodb to ensure good version is installed each time.

### CODE REVIEW

**Comment by pintojeremy** [comment]
Even after a few tries, I didn't manage to get tests to work.
I attempt to create an updated docker image for ubuntu bionic and ansible v 2.9.x but weirdly mongo wouldn't install fully, the service was missing..

File: tasks/main.yml [hunk 1]
```diff
@@ -1,103 +1,123 @@
...
 
 - name: apt update cache
   apt:
     update_cache: yes
   when: mongodb_repo_added.changed
   tags:
-  - mongo
+    - mongo
+
+- name: uninstall mongodb
```
Comment (SebastienElet) [inline tasks/main.yml:90]: This step may be really dangerous

**Comment by SebastienElet** [comment]
> Even after a few tries, I didn't manage to get tests to work.
> I attempt to create an updated docker image for ubuntu bionic and ansible v 2.9.x but weirdly mongo wouldn't install fully, the service was missing..

It looks like a packaging issue. I think we should create the systemd service with ansible.

File: tasks/main.yml (hunk 1 above)
Comment (pintojeremy) [reply tasks/main.yml:90]: How so? Do you think apt would remove data when removing mongodb ?

**Comment by pintojeremy** [comment]
> It looks like a packaging issue. I think we should create the systemd service with ansible.

Yeah. I'm a little bit confused about ubuntu "service" system between systemd and upstart. I tried to create a new docker image from `ubuntu:18.04` and even with systemd installed at build step, I couldn't get the handler to work..

**Review by GaetanCathelain — APPROVED** [review]

**Pull request merged**